.gitignore

```diff
@@ -57,6 +57,7 @@ phpunit.xml
 
 # Composer
 /vendor/
+/vendor-dist/
```

Comment:
Can we add this as an exclude pattern in `phpcs.xml.dist`?

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/3330/commits/5b42ac353ebadb4d64c777b938659f70a6570740 ✅